### PR TITLE
feat: add openstack_identity_group_v3 resource

### PR DESCRIPTION
**Pull request by c35sys**
This PR allows to create openstack_identity_group_v3 resource, referring to #895

### CODE REVIEW

File: openstack/resource_openstack_identity_group_v3_test.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+		},
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckIdentityV3GroupDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccIdentityV3Group_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckIdentityV3GroupExists("openstack_identity_group_v3.group_1", &group),
+					resource.TestCheckResourceAttrPtr(
+						"openstack_identity_group_v3.group_1", "name", &group.Name),
```
Comment (ozerovandrei) [inline openstack/resource_openstack_identity_group_v3_test.go:27]: It's better to check if the name is "group_1" here.

File: openstack/resource_openstack_identity_group_v3_test.go [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+				),
+			},
+			{
+				Config: testAccIdentityV3Group_update,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckIdentityV3GroupExists("openstack_identity_group_v3.group_1", &group),
+					resource.TestCheckResourceAttrPtr(
+						"openstack_identity_group_v3.group_1", "name", &group.Name),
```
Comment (ozerovandrei) [inline openstack/resource_openstack_identity_group_v3_test.go:35]: It's better to check if the name is "group_2" here so we will be sure that name was changed.

File: openstack/resource_openstack_identity_group_v3_test.go (hunk 1 above)
Comment (c35sys) [reply openstack/resource_openstack_identity_group_v3_test.go:27]: It is the case with `Config: testAccIdentityV3Group_basic` which is using `const testAccIdentityV3Group_basic`, `name` is the group name attribute.

File: openstack/resource_openstack_identity_group_v3_test.go (hunk 2 above)
Comment (c35sys) [reply openstack/resource_openstack_identity_group_v3_test.go:35]: Same thing it is the case with `Config: testAccIdentityV3Group_update` which is using `const testAccIdentityV3Group_update`.

File: openstack/resource_openstack_identity_group_v3.go [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+
+			"domain_id": {
```
Comment (kayrus) [inline openstack/resource_openstack_identity_group_v3.go:29]: According to keystone api ref

> The ID of the new domain for the group. The ability to change the domain of a group is now deprecated, and will be removed in subsequent release. It is already disabled by default in most Identity service implementations.

therefore please add `ForceNew: true,` to `domain_id`.

File: openstack/resource_openstack_identity_group_v3.go [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
```
Comment (kayrus) [inline openstack/resource_openstack_identity_group_v3.go:39]: Can you add an optional description field?

File: openstack/resource_openstack_identity_group_v3.go (hunk 3 above)
Comment (c35sys) [reply openstack/resource_openstack_identity_group_v3.go:29]: done

File: openstack/resource_openstack_identity_group_v3.go (hunk 4 above)
Comment (c35sys) [reply openstack/resource_openstack_identity_group_v3.go:45]: done

Comment (kayrus) [reply openstack/resource_openstack_identity_group_v3.go:45]: description is not reflected in the update function.

Comment (c35sys) [reply openstack/resource_openstack_identity_group_v3.go:45]: done

File: website/docs/r/identity_group_v3.html.markdown [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+resource "openstack_identity_group_v3" "group_1" {
+  name = "group_1"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - The name of the group.
```
Comment (kayrus) [inline website/docs/r/identity_group_v3.html.markdown:28]: I'm sorry to bother you again, but could you please reflect all the code changes you made by my request in the documentation? Thanks for your patience :)

Comment (c35sys) [reply website/docs/r/identity_group_v3.html.markdown:28]: no worries, I'll do that

Comment (c35sys) [reply website/docs/r/identity_group_v3.html.markdown:29]: done

**Pull request merged**